docs/man-pages/condor_watch_q.rst

```diff
@@ -112,69 +35,60 @@ General Options
 Tracking Options
 ''''''''''''''''
 
-These options control which jobs **condor_watch_q** will track,
-and how it discovers them.
+The following options control which jobs are tracked and how tracked
+jobs are discovered.
 
- **-users USER [USER ...]**
-    Choose which users to track jobs for.
-    All of the user's jobs will be tracked.
-    One or more user names may be passed.
+ **-users** *USER [USER ...]*
+    Specify which user(s) jobs to track.
 
- **-clusters CLUSTER_ID [CLUSTER_ID ...]**
-    Which cluster IDs to track jobs for.
-    One or more cluster ids may be passed.
+ **-clusters** *CLUSTER_ID [CLUSTER_ID ...]*
+    Specify which :ad-attr:`ClusterId`\'s to track.
 
- **-larger-than CLUSTER_ID**
-    Track jobs for all cluster IDs greater than
-    or equal to the specified *CLUSTER_ID*.
-    Note: This option does not discover how to track jobs.
+ **-larger-than** *CLUSTER_ID*
+    Track jobs for all :ad-attr:`ClusterId`\'s than or equal to the
+    specified *CLUSTER_ID*.
 
- **-files FILE [FILE ...]**
-    Which job event log files (i.e., the ``log`` file from ``condor_submit``)
-    to track jobs from.
-    One or more file paths may be passed.
+    .. note::
+        This option does not discover how to track jobs.
```

Comment:
Is there a better wording for this?  I find it confusing.

Reply:
I am just going to remove it